server/utilities/appointmentConfirmationUtilities.class.php

```diff
@@ -5,8 +5,6 @@
 class AppointmentConfirmationUtilities {
 
 	public static function generateAppointmentConfirmation($appointmentId) {
-		// TODO: THIS SHOULD NOT BE HARD-CODED TO TRUE AFTER VIRTUAL APPOINTMENTS STOP
```

Comment:
Love to see the TODOs getting deleted. Great change to the isVirtual field!

Reply:
Yah it feels good to cross one of these off haha